app/views/self_service/onboarding/_firms_index.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if principal.next_onboarding_action == :complete_a_firm %>
+  <div class="onboarding-message callout">
+    <%= paragraphs t('self_service.onboarding.firms_index.complete_a_firm_callout') %>
+  </div>
+<% end %>
+<% if principal.next_onboarding_action == :complete_an_adviser %>
+  <div class="onboarding-message callout">
+    <%= paragraphs t('self_service.onboarding.firms_index.complete_an_adviser_callout') %>
+    <p>
+      <% firm = principal.main_firm_with_trading_names.registered.first %>
+      <%= link_to t('self_service.onboarding.firms_index.complete_an_adviser_button', firm_name: firm.registered_name), new_self_service_firm_adviser_path(firm), class: 'button button--primary' %>
+    </p>
+  </div>
+<% end %>
```

Comment:
We considered it, but thought there was just enough difference between the partials to tell us that extracting would be more trouble than it was worth. The 'add adviser to xxx' buttons, for instance